### PR TITLE
add octopwnweb

### PR DESCRIPTION
**Pull request by marianomarciello**
This change add octopwnweb as requested in #3449.

### CODE REVIEW

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/octopwnweb/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+pkgver() {
+  cd $pkgname
+
+  # use, if no version string provided neither in sources nor in git describe:
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+  cd $pkgname
```
[the remark this answers is not in the document]
Comment (marianomarciello) [reply packages/octopwnweb/PKGBUILD:25]: How can I insert this dependencies ? WSNET can not be build without pip. Should I install WSNET as part of the installation process of octopwnweb ?

[the remark this answers is not in the document]
Comment (noraj) [reply packages/octopwnweb/PKGBUILD:25]: You should build a package for WSNET from https://github.com/skelsec/wsnet named `python-wsnet` following this template https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-setuptools and then add `python-wsnet` as dependency for octopwnweb, which will make two packages.

**Review by marianomarciello — COMMENTED** [review]
Hi I have made some change, hope now is better. Sorry for my mistakes but is my first time contributing.

**Comment by noraj** [comment]
Thx for contributing, I'll test your package soon

**Comment by noptrix** [comment]
thanks guys. @noraj any updates on progress here?

**Comment by noraj** [comment]
I'm working on it, it was far from ready.

@marianomarciello You forgot to check https://github.com/skelsec/wsnet/blob/144d5f13c8adfd2f9d793b34352731a852d1ce15/setup.py#L42-L43 in order to add the dependencies.

**Comment by noraj** [comment]
I also added a wrapper to start the web-server

**Review by noraj — APPROVED** [review]

**Pull request merged**